cartographer/requests/jsonapi_flask_request_mixin.py

```diff
@@ -9,8 +9,9 @@
 
 class JSONAPIFlaskRequestMixin(JSONAPIRequestInterface):
     def get_jsonapi_data(self):
-        if self.json and self.json.get('data'):
-            data = self.json.get('data')
+        json = self.get_json(self.mimetype == 'application/vnd.api+json')
```

Comment:
what is the first arg to `get_json` mean? can we pull the `self.mimetype == 'application/vnd.api+json'` check onto its own line then pass in that var as a named arg? i'm gueeeesssssinggg it means `force=X`, but this code could make that clearer